pages/_headers

```diff
@@ -0,0 +1,2 @@
+/*.rss
+  c  content-type: application/rss+xml; charset=utf-8
```

Comment:
From reading [the CloudFlare docs](https://developers.cloudflare.com/pages/configuration/headers/), shouldn't this be:
```suggestion
  Content-Type: application/rss+xml; charset=utf-8
```

Reply:
Testing that change out now on <https://pod.jonhoo.eu/podcast.rss> via https://github.com/jonhoo/pod-misc/commit/7ed9c636bd2334680acc298235d59ca08722bf69. Sadly, it doesn't seem to have worked :thinking:

Reply:
Ah, it's because Jekyll ignores files that start with `_`. Trying this now as a workaround: https://github.com/jonhoo/pod-misc/commit/fe9063fc784370bf83d16e46e3146a770c4a5149

Reply:
That did the trick! Will push a change like that here too.

Reply:
> From reading [the CloudFlare docs](https://developers.cloudflare.com/pages/configuration/headers/), shouldn't this be:

You're right of course. Not to myself: authoring PRs on the mobile gh website isn't a good idea 😅